Source/WebCore/rendering/RenderTextControlSingleLine.cpp

```diff
@@ -298,7 +298,7 @@ float RenderTextControlSingleLine::getAverageCharWidth()
     // of MS Shell Dlg, the default font for textareas in Firefox, Safari Win and
     // IE for some encodings (in IE, the default font is encoding specific).
     // 901 is the avgCharWidth value in the OS/2 table for MS Shell Dlg.
-    if (style().fontCascade().firstFamily() == "Lucida Grande")
+    if (style().fontCascade().firstFamily() == "Lucida Grande"_s)
```

Comment:
This code is super-obsolete. Lucida Grande hasn’t been the default font for anything for a long, long time.